clients/admin-ui/src/features/privacy-experience/form/OverlayForm.tsx

```diff
@@ -0,0 +1,89 @@
+import { Stack } from "@fidesui/react";
+
+import FormSection from "~/features/common/form/FormSection";
+import {
+  CustomSelect,
+  CustomTextArea,
+  CustomTextInput,
+} from "~/features/common/form/inputs";
+import { enumToOptions } from "~/features/common/helpers";
+import InfoBox from "~/features/common/InfoBox";
+import { BANNER_ENABLED_MAP } from "~/features/privacy-experience/constants";
+import { BannerEnabled } from "~/types/api";
+
+import PrivacyPolicy from "./PrivacyPolicy";
+
+const BANNER_ENABLED_OPTIONS = enumToOptions(BannerEnabled).map((opt) => ({
+  ...opt,
+  label: BANNER_ENABLED_MAP.get(opt.value) ?? opt.label,
+}));
+
+const INFO_TEXT =
+  "Configure your notice only banner, your consent banner and your privacy preferences below. It is good practice to complete all fields regardless of whether you are showing a notice only banner or an opt-in/opt-out banner.";
+
+const OverlayForm = () => (
+  <Stack spacing={6}>
+    <InfoBox text={INFO_TEXT} />
+    <FormSection
+      title="Cookie banner and privacy preferences labeling"
+      data-testid="banner-and-preferences-labeling"
+    >
+      <CustomTextInput
+        name="title"
+        label="Banner title"
+        variant="stacked"
+        isRequired
+      />
+      <CustomTextArea
+        label="Banner description"
+        name="description"
+        variant="stacked"
+        isRequired
+      />
+      <CustomTextInput
+        label="Accept button displayed on the Banner and “Privacy preferences”"
+        name="accept_button_label"
+        variant="stacked"
+        isRequired
+      />
+      <CustomTextInput
+        label="Reject button displayed on the Banner and “Privacy preferences”"
+        name="reject_button_label"
+        variant="stacked"
+        isRequired
+      />
+      <CustomTextInput
+        label="“Privacy preferences” link label"
```

Comment:
Should this one be required as well? It looked like it's not required on the backend

Reply:
Oh wait it is actually required for overlays not privacy centers, is this something required for both you think?

Reply:
ah I see, yeah I think it makes sense to be required for the overlay but not for the privacy center. I can make that change in the FE, and then the BE can stay the same?

Reply:
sounds good!